presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -479,9 +479,12 @@ else if (!partitionColumnNames.contains(name)) {
         table.setPartitionKeys(partitionColumns);
         table.setSd(sd);
 
-        PrivilegeGrantInfo allPrivileges = new PrivilegeGrantInfo("all", 0, tableOwner, PrincipalType.USER, true);
+        PrivilegeGrantInfo selectPrivilege = new PrivilegeGrantInfo("select", 0, tableOwner, PrincipalType.USER, true);
```

Comment:
Perhaps you misunderstood me. I wanted to know when was this change of *removing* ALL privileges introduced in Hive. Can you please point me to a Hive document or release notes that mention about *not* supporting ALL privileges? Or has this not yet been documented in Hive, but has been implemented?